test/k8sT/manifests/demo_ds_local.yaml

```diff
@@ -0,0 +1,233 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testds
+spec:
+  selector:
+    matchLabels:
+      zgroup: testDS
+  template:
+    metadata:
+      labels:
+        zgroup: testDS
+    spec:
+      containers:
+      - name: web
+        image: docker.io/cilium/echoserver:1.10.1
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 80
+      - name: udp
+        image: docker.io/cilium/echoserver-udp:v2020.01.30
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 69
+          protocol: UDP
+      terminationGracePeriodSeconds: 0
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node.cloudprovider.kubernetes.io/uninitialized
+        value: "true"
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testclient
+spec:
+  selector:
+    matchLabels:
+      zgroup: testDSClient
+  template:
+    metadata:
+      labels:
+        zgroup: testDSClient
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: web
+        image: docker.io/cilium/demo-client:latest
+        imagePullPolicy: IfNotPresent
+        command: [ "sleep" ]
+        args:
+          - "1000h"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-k8s2
+spec:
+  selector:
+    matchLabels:
+      zgroup: test-k8s2
+  template:
+    metadata:
+      labels:
+        zgroup: test-k8s2
+    spec:
+      containers:
+      - name: web
+        image: docker.io/cilium/echoserver:1.10.1
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
+          hostPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 80
+      - name: udp
+        image: docker.io/cilium/echoserver-udp:v2020.01.30
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 69
+          hostPort: 6969
+          protocol: UDP
+      terminationGracePeriodSeconds: 0
+      nodeSelector:
+        "cilium.io/ci-node": k8s1
```

Comment:
Note that the only diff with the standard `demo_ds.yaml` here is the `nodeSelector`. If we had better templating, this extra file wouldn't be necessary but I figured this is the easiest way to get this working right now.